doc/faq/a06-global-build-options.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
```

Comment:
> /home/runner/work/Arduino/Arduino/doc/faq/a06-global-build-options.rst: WARNING: document isn't included in any toctree

Reply:
I maintain this file as a `.md` on my machine and use `pandoc` to turn it into a `.rst` for a PR. So far I have noticed the `:orphan:` line seemed to magically appear later after a merge, 

Are you indicating that this `:orphan:` line should always be added?

Reply:
Missed c/p
https://github.com/esp8266/Arduino/actions/runs/4097689448/jobs/7066241098#step:4:222

Sphinx build thinks this is something worth warning about
#8686 opted out to use this tag, but using this page in a toctree somewhere would fix it
(also see other faq/*.rst)